lib/sles4sap_publiccloud.pm

```diff
@@ -363,9 +363,13 @@ sub check_takeover {
 
   TAKEOVER_LOOP: while (1) {
         my $topology = $self->get_hana_topology();
+        record_info("Cluster topology", $topology);
         $retry_count++;
         for my $entry (@$topology) {
             my %host_entry = %$entry;
+            record_info("Cluster Host", $host_entry{vhost});
+            record_info("Cluster Host", $host_entry{sync_state});
+            record_info("Cluster Host", $hostname);
```

Comment:
Since you are recording `$host_entry{vhost}` and `$host_entry{sync_state}`, wouldn't it make more sense if the 3 `record_info`s were after lines 
```
            die "Missing 'vhost' field in topology output" unless defined($host_entry{vhost});
            die "Missing 'sync_state' field in topology output" unless defined($host_entry{sync_state});
 ```
 ?

Reply:
yes make sense thx